chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -25,27 +25,50 @@
 
 
 Zotero.QuickCopy = new function() {
-	this.getFormattedNameFromSetting = getFormattedNameFromSetting;
-	this.getSettingFromFormattedName = getSettingFromFormattedName;
-	this.getContentType = getContentType;
-	this.stripContentType = stripContentType;
-	this.getFormatFromURL = getFormatFromURL;
-	this.getContentFromItems = getContentFromItems;
 	
 	var _initialized = false;
 	var _formattedNames = {};
 	
+	/*
+	 * Return Quick Copy setting object from string
+	 */
+	this.unserializeSetting = function (setting) {
+		var settingObject = {};
+		
+		if (typeof setting === 'string' || setting instanceof String) {
+			try {
+				settingObject = JSON.parse(setting);
+			} catch (e) {
+				var parsedSetting = setting.match(/(bibliography|export)(?:\/([^=]+))?=(.+)$/);
+				if (parsedSetting) {
+					settingObject.mode = parsedSetting[1] ? parsedSetting[1] : '';
+					settingObject.contentType = parsedSetting[2] ? parsedSetting[2] : '';
```

Comment:
`parsedSetting[2]` could be `null` though, so that's ok, but you can do `settingObject.contentType = parsedSetting[2] || ''` to make this look cleaner.